src/types/Vendors.ts

```diff
@@ -21,54 +22,64 @@ export const BINDER_POS_VENDORS: Record<string, BinderPosBackendVendor> = {
     "gufgames.myshopify.com": {
         backend: VendorBackend.BINDER_POS,
         url: "gufgames.myshopify.com",
+        vendorUrl: "https://guf.com.au",
         name: "Guf"
     },
     "the-hall-of-heroes.myshopify.com": {
         backend: VendorBackend.BINDER_POS,
         url: "the-hall-of-heroes.myshopify.com",
+        vendorUrl: "https://thehallofheroes.com.au",
         name: "The Hall of Heroes"
     },
     "cracking-singles.myshopify.com": {
         backend: VendorBackend.BINDER_POS,
         url: "cracking-singles.myshopify.com",
+        vendorUrl: "https://cracking-tcgsingles.com.au",
         name: "Cracking Singles"
     },
     "next-level-games-ringwood.myshopify.com": {
         backend: VendorBackend.BINDER_POS,
         url: "next-level-games-ringwood.myshopify.com",
+        vendorUrl: "https://gamesportal.com.au",
         name: "Games Portal"
     },
     "good-games-townhall.myshopify.com": {
         backend: VendorBackend.BINDER_POS,
         url: "good-games-townhall.myshopify.com",
+        vendorUrl: "https://tcg.goodgames.com.au",
         name: "Good Games National"
     },
     "good-games-morley.myshopify.com": {
         backend: VendorBackend.BINDER_POS,
         url: "good-games-morley.myshopify.com",
+        vendorUrl: "https://goodgamesmorley.com.au",
         name: "Good Games Morley"
     },
     "good-games-cannington.myshopify.com": {
         backend: VendorBackend.BINDER_POS,
         url: "good-games-cannington.myshopify.com",
+        vendorUrl: "https://good-games-cannington.myshopify.com",
         name: "Good Games Cannington"
     },
     "good-games-adelaide-sa.myshopify.com": {
         backend: VendorBackend.BINDER_POS,
         url: "good-games-adelaide-sa.myshopify.com",
+        vendorUrl: "https://ggadelaide.com.au",
         name: "Good Games Adelaide"
     },
     "unplugged-games.myshopify.com": {
         backend: VendorBackend.BINDER_POS,
         url: "unplugged-games.myshopify.com",
+        vendorUrl: "https://tcgsingles.com.au",
         name: "Unplugged Games"
     }
 }
 export const MTGMATE_VENDORS: Record<string, MtgMateVendor> = {
     "mtgmate.com.au": {
         backend: VendorBackend.MTGMATE,
         name: "MTGMate",
-        url: "https://polluxus.dev/mtgmate/?utf8=✓&decklist="
+        url: "https://polluxus.dev/mtgmate/?utf8=✓&decklist=",
```

Comment:
You probably have already seen from using it, but this URL is a simple NGINX proxy to MTGMate - IIRC, I had issues with CORS when trying to fetch an MTGMate page directly. Here's the NGINX config used:

```nginx
location /mtgmate/ {
    resolver 8.8.8.8;
    proxy_ssl_server_name on;
    proxy_redirect off;
    proxy_set_header Host www.mtgmate.com.au;
    proxy_pass https://www.mtgmate.com.au/cards/decklist_results$is_args$args;
    add_header Access-Control-Allow-Origin *;
}
```

Reply:
yeah I saw that. We can keep using this to source/fetch cards as it's easier to parse than the new website page.
That's the main reason I've added vendorUrl so we can redirect users to the "real" page